sections/06_discussion.md

```diff
@@ -475,17 +475,12 @@ sharing and analysis [@doi:10.1038/ng.3830].
 The sharing of high-quality, labeled datasets will be especially valuable.  In
 addition, researchers who invest time to preprocess datasets to be suitable for
 deep learning can make the preprocessing code (e.g. Basset
-[@tag:Kelley2016_basset] and variationanalysis [@tag:Torracinta2016_deep_snp])
```

Comment:
My mistake, should be fixed here: https://github.com/greenelab/deep-review/pull/577/commits/820bfbe96d271f5d69e180938c65621b19c5a2e6